## Time stamped drive command for robots with Ackermann steering.
#  $Id: AckermannDriveStamped.msg 1958 2012-02-11 02:35:54Z jack.oquin $

Header          header
AckermannDrive  drive
